src/core/mainloop/periodic.c

```diff
@@ -24,6 +32,11 @@
  */
 static const int MAX_INTERVAL = 10 * 365 * 86400;
 
+/**
+ *
+ **/
```

Comment:
Some documentation got truncated? :)

Reply:
Thanks; I've added a commit to fix this.